pulsar-client-cpp/docker-lib-check.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+if ! dpkg -l libsnappy-dev; then
+    apt-get install -y libsnappy-dev
```

Comment:
@yittg can you add the library to https://github.com/apache/pulsar/blob/master/build/docker/Dockerfile

Reply:
As I said, it's in [Dockerfile](https://github.com/apache/pulsar/blob/41bc407e1642aec3805d0f077694edd835b3ea1c/build/docker/Dockerfile#L31), but the docker image with tag `ubuntu-16.04` is outdated which has no full libs installed.